streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -319,7 +321,7 @@ public void shouldNotThrowUnsupportedOperationExceptionWhenInitializingStateStor
                 new PartitionInfo(changelogName, 0, Node.noNode(), new Node[0], new Node[0])));
         final KStreamBuilder builder = new KStreamBuilder();
         builder.stream("topic").groupByKey().count("my-store");
-        final ProcessorTopology topology = builder.build(0);
+        final ProcessorTopology topology = builder.setApplicationId(applicationId).build(0);
```

Comment:
Explained as above. We are now solely depending on the topology builder to give us the information about which state stores should be changelog enabled and what are their changelog topic names. Before that we have a separate `loggingEnabled` parameter in register to indicate that.